components/tooltip/package.json

```diff
@@ -17,16 +17,16 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.21",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^3.0.0"
   },
   "devDependencies": {
     "@spectrum-css/component-builder": "^3.2.0",
```

Comment:
Let's use `@spectrum-css/component-builder-simple`, which is now at `^2.0.0`, please.

Reply:
Totally overlooked this detail, thanks. Good to know the newest version. Also.
✔ Done